README.md

```diff
@@ -7,13 +7,13 @@ Visualization tools for the Sea Ice Indexes
 ### Demo
 
 * **Notebook:**
-    
+
    https://mybinder.org/v2/gh/metno/sea-ice-index-viz/main?labpath=bokeh-app%2FSIE.ipynb
 
 
 * **Bokeh APP:**
 
-   https://mybinder.org/v2/gh/metno/sea-ice-index-viz/main?urlpath=/proxy/5006/bokeh-app    
+   https://mybinder.org/v2/gh/huaracheguarache/sea-ice-index-viz/legend_test_hide?urlpath=/proxy/5006/bokeh-app
```

Comment:
to merge the PR this line should point to the main repository, not your fork/branch

Reply:
Ok, done =)